docs/WifiPerformance.md

```
@@ -61,76 +61,77 @@ All wireless adapters were tested under consistent conditions - each positioned
 
 ## WiFi performance
 
-_Measured 2026-07-01 06:33 UTC · [run](https://github.com/armbian/autotests/actions/runs/28497549738)_
+_Measured 2026-07-01 11:47 UTC_
 
-**47** wireless link(s) · ✅ **47** pass · 📶 peak ↑ **756** / ↓ **643** Mbps
+**48** wireless link(s) · ✅ **48** pass · 📶 peak ↑ **681** / ↓ **868** Mbps
 
 #### IEEE 802.11ac
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)